test/integration/util/extension.go

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
s/2018/2021/? how much have you changed? it's hard to see the diff in a new file.

Reply:
done. I've changed very little, but I guess there's no harm in keeping the copyright up to date.